app/models/user.rb

```diff
@@ -3,4 +3,10 @@ class User < ApplicationRecord
   validates_uniqueness_of :email
 
   has_many :buddies
+  has_many :feelings
+  has_many :feeling_categories
+  has_one :music_preference
+  has_one :media_preference
+  has_one :activity_preference
+  has_one :time_preference
```

Comment:
Can a user have more than one of each of these preferences?

Reply:
@freeheeling no...it has one preference object that has many attributes